src/rajAJun20/StaticNonStatic.java

```diff
@@ -0,0 +1,41 @@
+package rajAJun20;
+
+public class StaticNonStatic {
+			double sum(double var1, double var2, double var3) {
+	    	double sum = var1 + var2 + var3;
+	    	System.out.println("Sum of three variable is: " + sum);
+	    	return sum;
+	    }
+	    double average(double var1, double var2, double var3) {
+	    	double average = (var1 + var2 + var3)/3;
+	    	sum(var1, var2, var3);
```

Comment:
no need of line 11.
line number 10 is not correct. you should write
double average = sum(var1 ,var2,var3)/3;

Reply:
Done Suggested changes....!!!